dev-docs/RFCs/v2.1/normalize-3d-tile-loaders.md

```diff
@@ -0,0 +1,103 @@
+Although the specs are distinct from each other, they do share the fundamental components in common. Both specifications divide massive dataset into small and renderable chunks that structured in a hierarchical tree.
+Child tiles in the tree are subdivisions of their parents and have finer details, such that less data can be visualized by only loading and drawing the visible regions and level of details can be dynamically
+displayed by zoom level.
+
+- Dynamically loading the dataset based on viewport requires tree traversing and strategies to decide whether a specific tile node is sufficient for current viewport (geo boundaries, zoom).
+- The math needed for loading is common in either graphics or geospatial, i.e. Bounding Box, frustum culling, Ellipsoid, and coordinate transformations, etc.
+- To provide smooth user interaction with the map (panning, zooming), requires frequently loading and unloading tiles, and thus request scheduling and caching are needed to manage tile resources efficiently.
+- A unified parsed format for 3d tiles category will empower all the rendering frameworks to load different 3D tiles specifications.
+
+Therefore this RFC joins the 3D tiles and i3s loaders and specifies the semantics of the unified output format. This would also potentially benefit loading other 3D standards, i.e. Potree.
+
+## Concepts
+
+- [OGC 3D Tiles](https://www.opengeospatial.org/standards/3DTiles) standard
+- [OGC i3s](https://www.opengeospatial.org/standards/i3s) standard
+- **Tile Header Hierarchy** - An initial, "minimal" set of data listing the _hierarchy of available tiles_, with minimal information to allow an application to determine which tiles need to be loaded based on a certain viewing position in 3d space.
+- **Tile Header** - A minimal header describing a tiles bounding volume and a screen space error tolerance (allowing the tile to be culled if it is distant), as well as the URL to load the tile's actual content from.
+- **Tile Cache** - Since the number of tiles in big tilesets often exceed what can be loaded into available memory, it is important to have a system that releases no-longer visible tiles from memory.
+- **Tileset Traversal** - Dynamically loading and rendering 3D tiles based on current viewing position, possibly triggering loads of new tiles and unloading of older, no-longer visible tiles.
+
+## Components
+
+- [`Tileset3DLoader`](docs/api-reference/3d-tiles/tileset-3d-loader): Load a tileset into a json object.
+- [`Tileset3D`](docs/api-reference/3d-tiles/tileset-3d): Class instanced from a tileset object.
+- [`Tile3DLoader`](docs/api-reference/3d-tiles/tile-3d-loader): Load a tile node in a tileset.
+- [`Tile3DHeader`](docs/api-reference/3d-tiles/tile-3d-header): Class instanced from a tile header.
+
+## Usage
+
+1. Load a tileset
+
+```js
+import {Tileset3DLoader, Tileset3D} from '@loaders.gl/3d-tiles';
+import {load} from '@loaders.gl/core';
+
+const tilesetUrl = 'A tileset url';
+
+// load 3d tiles from ion
```

Comment:
Maybe `3DTilesTileSetLoader` since the idea is various 3d tile formats could be supported and it would be similar to `I3STileSetLoader` in the naming convention